### PR TITLE
visual refresh - color tokens

### LINKED ISSUE #237: New visual design color tokens
## Description:

Add in color tokens for the [new visual design](https://www.figma.com/design/Iuw9me6kYftBF4WTCEsCZz/Query-Connector?node-id=68-9738&m=dev)

## Acceptance Criteria:

### PR DESCRIPTION
**Pull request by fzhao99**
# PULL REQUEST

## Summary

Adding / applying new color tokens to match the [new visual designs](https://www.figma.com/design/Iuw9me6kYftBF4WTCEsCZz/Query-Connector?node-id=68-9738&m=dev)

## Related Issue

Fixes #237 

### For design
General alignment audit to make sure new colors / design tweaks match [Figma](https://www.figma.com/design/Iuw9me6kYftBF4WTCEsCZz/Query-Connector?node-id=68-9738&m=dev)

## Checklist
- [x] Descriptive Pull Request title
- [x] Link to relevant issues
- [x] Provide necessary context for design reviewers

### CODE REVIEW

File: query-connector/src/app/query/components/CustomizeQuery.tsx [hunk 1]
```diff
@@ -72,13 +72,15 @@ const CustomizeQuery: React.FC<CustomizeQueryProps> = ({
...
 
   // Compute counts of each tab-type
-  const countLabs = countDibbsConceptTypeToVsMapItems(valueSetOptions.labs);
-  const countConditions = countDibbsConceptTypeToVsMapItems(
-    valueSetOptions.conditions,
-  );
-  const countMedications = countDibbsConceptTypeToVsMapItems(
-    valueSetOptions.medications,
-  );
+  const countLabs = valueSetOptions.labs
```
Comment (fzhao99) [inline query-connector/src/app/query/components/CustomizeQuery.tsx:75]: there was a null pointer error here that occurred when there were empty fetched concept types, so added these checks to have the page render correctly.

File: query-connector/src/app/query/components/stepIndicator/TitleBox.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { Mode } from "@/app/constants";
+import { PAGE_TITLES } from "./StepIndicator";
+
+type TitleBoxProps = {
+  step: Mode;
+};
+
+/**
+ * TitleBob that renders the page title and step counter state
```
Comment (nickclyde) [inline query-connector/src/app/query/components/stepIndicator/TitleBox.tsx:9]: Good idea to move this into it's own component! Small typo here in the component docs

File: query-connector/src/styles/_variables.scss [hunk 3]
```diff
@@ -5,22 +5,170 @@ $query-connector-max-width: 45rem;
...
+
+$table-row-border: $gray-300;
+$table-header: $gray-300;
+
+$stepper-completed: $blue-500;
+
+$dropdown-hover: $gray-200;
+
+$input-field-border: $gray-500;
+$checkbox-border: $gray-500;
```
Comment (nickclyde) [inline query-connector/src/styles/_variables.scss:171]: Wowza that's a lotta color variables!! Great work

**Review by nickclyde — APPROVED** [review]
Awesome work Bob! Just ran it locally, everything looks really nice!

**Pull request merged**